mediapipe/calculators/ovms/modelapiovmsadapter.cc

```diff
@@ -117,6 +119,9 @@ InferenceOutput OVMSInferenceAdapter::infer(const InferenceInput& input) {
             OVMS_BUFFERTYPE_CPU,
             NOT_USED_NUM));
     }
+#if (OVMS_DUMP_TO_FILE == 1)
```

Comment:
This file will be moved at some point to model_api repository, so I am not sure if we should leave it here in addition to inference calculator. On the other hand for now it is the only way to debug geti graphs.